src/Illuminate/Console/Command.php

```diff
@@ -506,9 +507,11 @@ public function warn($string, $verbosity = null)
      */
     public function alert($string)
     {
-        $this->comment(str_repeat('*', strlen($string) + 12));
+        $cleanString = strip_tags($string);
```

Comment:
Since `cleanString` is only used in length calculations, why not compute the length here?

Reply:
Because it's used two times.Or this in not problem?

Reply:
Yeh, and the variable name of clean string doesn't really indicate what it is. `$length = ...` is much clearer.

Reply:
Huh... you're right.